conf/messages

```diff
@@ -1124,7 +1124,7 @@ tai.taxFreeAmount.table.taxComponent.HotelAndMealExpenses=Hotel and meal expense
 tai.taxFreeAmount.table.taxComponent.IncapacityBenefit=Incapacity Benefit
 tai.taxFreeAmount.table.taxComponent.IncomeTaxPaidButNotDeductedFromDirectorsRemuneration=Income Tax paid but not deducted from director’s remuneration
 tai.taxFreeAmount.table.taxComponent.JobExpenses=Job expenses
-tai.taxFreeAmount.table.taxComponent.JobSeekersAllowance=Jobseeker’s Allowance
+tai.taxFreeAmount.table.taxComponent.JobSeekersAllowance=Jobseeker''s Allowance
```

Comment:
The test fails from an unescaped quote, but when I put two apostrophes it displays one and the tests pass. Is this the best way to do it?

Reply:
`\'` is preferred